llvm/test/Transforms/CorrelatedValuePropagation/icmp.ll

```diff
@@ -1571,8 +1567,7 @@ define void @test_icmp_trunc(i8 %a) {
 ; CHECK-NEXT:    [[CMP1:%.*]] = icmp ne i8 [[A:%.*]], 0
 ; CHECK-NEXT:    br i1 [[CMP1]], label [[IF_TRUE:%.*]], label [[IF_FALSE:%.*]]
 ; CHECK:       if.true:
-; CHECK-NEXT:    [[TRUNC:%.*]] = trunc i8 [[A]] to i1
-; CHECK-NEXT:    call void @check1(i1 [[TRUNC]])
+; CHECK-NEXT:    call void @check1(i1 true)
```

Comment:
this is not correct.

Reply:
yes sorry hade a copy past error in the proof that made me confused.
have removed the faulty transform